src/manifest.json

```diff
@@ -27,7 +27,8 @@
     "storage",
     "tabs",
     "webRequestBlocking",
-    "webRequest"
+    "webRequest",
+    "proxy"
```

Comment:
If you are adding the proxy permission, which is an [Advised Permission](https://extensionworkshop.com/documentation/develop/request-the-right-permissions/#advised-permissions), you might also want to add 'bookmarks' and remove the [approval request](https://github.com/mozilla/multi-account-containers/blob/4b56a2f0bb91cd9a12b95121fcff0c8e96ff66bd/src/js/options.js#L6) in `options.js`

Reply:
Filed this as a seperate issue: https://github.com/mozilla/multi-account-containers/issues/2138